lib/commands/vms.js

```diff
@@ -32,8 +32,13 @@ exports.handler = async argv => {
         return;
     }
 
-    let transformed = vms.reduce((table, {id, ...x}) => {
+    let transformed = await vms.reduce(async (table, {id, ...x}) => {
         table[id] = x;
+
+        const name = x["name"];
+        const health = provider ? await providers[provider].health(name) : await providers[x["provider"]].health(name);
+        table[id]["online"] = health;
+
```

Comment:
I don't like how this code is written if anyone has tips to improve it.

Reply:
I see, this is compact but might be a little harder to understand. What do you think of [this implementation](https://github.com/ottomatica/Baker/blob/3f23e3ef64747e68982dab9c6d293708d4e61a12/lib/modules/providers/virtualbox.js#L34-L51) in [Baker](https://github.com/ottomatica/baker) which has a similar functionality?